sql/updates/world/master/9999_99_99_00_world_dragonriding.sql

```diff
@@ -0,0 +1,18 @@
+delete from spell_script_names WHERE ScriptName IN (
+"spell_af_skyward_ascent",
+"spell_af_energy",
+"spell_af_surge_forward",
+"spell_af_whirling_surge"
+);
+INSERT INTO spell_script_names VALUES
+(374763, "spell_af_skyward_ascent"),
+(372610, "spell_af_skyward_ascent"),
+(372773, "spell_af_energy"),
+(372608, "spell_af_surge_forward"),
+(361584, "spell_af_whirling_surge");
+
+DELETE FROM `spell_linked_spell` WHERE spell_trigger = 406095 AND spell_effect = 423624;
+DELETE FROM `spell_linked_spell` WHERE spell_trigger = 372773 AND spell_effect = 433547;
+INSERT INTO `spell_linked_spell` VALUES
```

Comment:
`spell_linked_spell` is a table we no longer use and is scheduled for dropping at some point. Please use a SpellScript instead